playbooks/technical-planning.md

```diff
@@ -0,0 +1,81 @@
+1. Changes affecting other teams' products or engineers' workflows
+1. New integrations or services
+
+If the feature you're working on follows our best-practices and doesn't require technical decisions larger than what would fit in a single PR, it's OK to skip.
+
+If you're working on a small feature or it doesn't fit the criteria above but you still want to write one, that's fine! Feel free to keep it small.
+
+## Tech Plans vs. Product Briefs
+Technical plans discuss and propose a technical implementation, while product briefs should govern the design, user experience, and business requirements of a feature. We recommend linking to an associated product brief for context, but product and requirement decisions should live on the product brief, not on the tech plan.
+
+It's natural to discover new product implications through the tech planning process. If this happens, make sure those discussions are routed to the product brief for clarity.
+
+If product-related feedback comes up while the tech plan is being reviewed, that is often a sign that there is more to discuss and debate at the product brief-level. If you are authoring a tech plan and this happens, it is recommended that you bring the conversation immediately to the associated PM(s). Once the question or feedback is addressed, the tech planning can continue.
+
+## Tech Plan States
+### Draft
+A tech plan in this state is a work in progress.
+
+### Proposed
+A tech plan in this state is officially open for feedback. Anyone should feel free to look at Proposed tech plans and leave their thoughts.
+
+### Accepted
+A tech plan in this state has been approved by all associated stakeholders. At this stage, it can be implemented or deprioritized.
+
+### Closed
+A tech plan in this state has either been rejected outright (i.e. it was proposing a new technology that the team decides not to adopt) or otherwise abandoned before it was able to be formally accepted.
+
+### Implemented
+A tech plan in this state has been implemented. Generally, tech plans can be put into this state at the same time that the associated product brief, if there is one, is moved to "Done".
+
+## Writing a tech plan
+We recommend following the template in Notion, but feel free to add or remove sections. Be as clear, concise, and organized as possible when writing so that it is easy for others to understand and leave quality feedback.
+
+If the technical plan involves a contentious or impactful technical decision, we recommend creating a separate, linked, document to capture the alternatives considered ([example here](https://www.notion.so/artsy/d17290484e6b40ac9e871eb7070dd3e8?v=81f6f3bf425b43e48fadd3c36adc2e09)). This allows the tech plan to focus on the chosen option.
+
+As with the tech planning process itself, creating decision matrices benefits from [collaboration and feedback](https://review.firstround.com/this-matrix-helps-growing-teams-make-great-decisions).
+
+## Stakeholders
```

Comment:
Could this be folded _into_ the stakeholders section? Like, ensure that at least one member of the affected team is added to that list (with a little explanation for why?). Totally open to adding a new section, just wondering how it would behave differently.